generators/app/templates/src/layouts/mainGuide.soy

```diff
@@ -0,0 +1,15 @@
+{namespace mainGuide}
```

Comment:
yes, we can. I just created this `mainGuide` template to not duplicate few lines. but it's really confusing. I'll change this